Pull request for Milestone 3 - Tic Tac Toe

bruna-genz opened the pull request:
In this milestone I created the file game_logic.rb in the bin repository. This file contains the classes and methods used to make the game work. Also, I edited the main.rb file to create a better interaction with the user.

Torres-ssf commented:
### Pull Request Under Review By Sérgio (TSE)

Torres-ssf commented on lib/game_logic.rb:
```diff
@@ -0,0 +1,138 @@
+  end
+
+  def draw?
+    turn >= 9
```
**[REQUIRED]**
We have a bug related to win and draw message. If a player won the match in the last possible move, both draw and winner messages appear:
![Screenshot from 2019-10-02 08-14-59](https://user-images.githubusercontent.com/30977845/66042371-a1d7be00-e4f2-11e9-9e00-c6806719891d.png)

Torres-ssf commented on lib/game_logic.rb:
```diff
@@ -0,0 +1,138 @@
+end
+
+class Player
+  attr_accessor :name, :mark
```
**[REQUIRED]**
 The player's `name` never changes after been assigned. You can safely use `attr_reader` for it.

Torres-ssf commented on lib/game_logic.rb:
```diff
@@ -0,0 +1,138 @@
+
+class Board
+  attr_accessor :board
+  attr_reader :mark
```
**[REQUIRED]**
After been assigned, the board's value never changes. We can change the encapsulation here making `board` attr_reader.

bruna-genz commented:
In the last commit, I fixed the bug from when a player won the match in the last possible move and changed the variables board and name to attr_reader.
